Datasets/LanguageModelDataset.swift

```diff
@@ -0,0 +1,135 @@
+import TensorFlow
+
+//Build a dataset suitable for language modeling from an array of texts
```

Comment:
Could you please use triple slashes for documentation comments?
Since types are nouns, the doc comment for a type should be a noun clause, not a verb clause (e.g. `Build`).

```suggestion
/// A language model dataset.
```

I found these [Swift API documentation comment guidelines](https://swift.org/documentation/api-design-guidelines/#write-doc-comment) to be very helpful. All declarations (especially public ones) can use a doc comment.

Reply:
I had triple slashes but the swift-formatter removed them FYI. Will address.